docs/sampling.md

```diff
@@ -0,0 +1,77 @@
+# Sampling in COMPAS
+
+------------
+
+Here are some basic instructions for efficient sampling of the COMPAS input parameters, using the python sampling package Stroopwafel.
+
+Note that the intended Stroopwafel functionality for "Adaptive Importance Sampling" is not yet implemented, but is currently in development.
+
+## Requirements
+
+--------------
+
+If you have not already, you will need to install Stroopwafel. If you have admin rights, Stroopwafel can be installed on your system with `pip install stroopwafel`. 
+
+
+## Instructions
+
+---------------
+
+To use Stroopwafel sampling, copy `defaults/stroopwafel_interface_default.py` into your working directory. 
+
+
+### Settings
+
+#### 1. PythonSubmit
+
+If you are running COMPAS on default settings, skip this section. 
+
+If you have many non-default COMPAS arguments, you may want to set them in a 
+`pythonSubmit.py` file in the same directory. The file must be named this way and placed
+in the same directory as the `stroopwafel_interface.py` file. 
+
+You can achieve this with `cp $COMPAS_ROOT_DIR/defaults/pythonSubmitDefault.py ./pythonSubmit.py`, 
+and setting your desired options.
+
+Then set the `usePythonSubmit` parameter to `True` in the `stroopwafel_interface.py`.
+
+#### 2. Stroopwafel inputs
+
+The lines below `usePythonSubmit` represent stroopwafel inputs. These are treated as
+defaults, but can be overriden by command-line arguments to stroopwafel. 
+See `python3 stroopwafel_interface.py --help`.
+
+`num_systems` is the total number of binaries you would like to evolve. 
+This value overrides the value set in the pythonSubmit.
+
+`num_cores` is the number of cores you would like to use to parallelize your run. More cores means your run will finish sooner, but may reduce your ability to run other tasks while you wait. On linux systems, the command `echo $(nproc)` will tell you how many (virtual) CPUs you have available.
+
+`num_per_core` is the number of systems to run on a core at a given time. This translates to the number of systems in a single batch file. This is more relevent for adaptive importance sampling.
+
+`mc_only` specifies if you would like to do naive MC sampling only. Currently, this option must be set to True
+
+`run_on_hpc` specifies if you are running on a High-Performance Computer (HPC). 
+If so, see [docs/compasHPC.md](compasHPC.md) for assistance.
+
+`output_folder` a string specifying the output folder. Relative paths will be appended onto the current directory path.
+
+`output_filename` a string specifying the name of the output samples file.
+
+`debug` for development purposes.
```

Comment:
Good to add here is that debug is specifically used to print the output / error that COMPAS will sendover, which is otherwise hidden. It has nothing to do with debugging stroopwafel.

Reply:
Oh really? I didn't that, thanks!